releases/release-1.17/README.md

```diff
@@ -60,7 +60,7 @@ The 1.17 release cycle is proposed as follows:
 | 1.17.0-beta.1 released | Branch Manager | Tue November 5 | 
 | Docs deadline - Open placeholder PRs | Docs Lead | Fri November 8 |
 | **Begin [Code Freeze]** (EOD PST) | Branch Manager | Thu November 14 | week 8 |
-| Docs deadline - PRs ready for review | Docs Lead | Fri November 15 | 
+| Docs deadline - PRs ready for review | Docs Lead | Tues November 19 |
```

Comment:
@daminisatya actually this should stay and the TL;DR [here](https://github.com/kubernetes/sig-release/pull/824/files#diff-57bffa687aa84ec643a46c76597f6ca3R28) read Nov.22, as for "Docs Complente", below.